keyboards/boardsource/3x4/config.h

```diff
@@ -6,7 +6,7 @@
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xF7E0
```

Comment:
```suggestion
#define VENDOR_ID       0x4273 // "Bs" - Boardsource
```

Reply:
should I take the time to change this for the other bourdsource boards - others aren't currently supported by VIA, so I'm assuming this wouldn't be a breaking change?